carla/recourse_methods/catalog/actionable_recourse/model.py

```diff
@@ -146,13 +146,21 @@ def get_counterfactuals(self, factuals: pd.DataFrame) -> pd.DataFrame:
         intercepts = self._intercepts
         action_set = self._action_set
 
+        factuals = (
+            factuals.reset_index()
+        )  # to keep matching indexes for iterrows and coeffs
```

Comment:
This is a bit weird formatted with the brackets. I guess its auto-formatted since you put the comment on the same line as the code

Reply:
Right, makes more sense maybe to put above. Will change this.